RecoParticleFlow/PFSimProducer/plugins/HGCalTrackCollectionProducer.cc

```diff
@@ -0,0 +1,190 @@
+// S. Zenz, 12 February 2015
+//
+// Splits a track collection into two, based on whether they propagate to the HGCal or not
+// Tracks with bad pt resolution (suspected fakes) are dropped and not in either collection
```

Comment:
I do not see any SIM dependence in this file.
Please move out of PFSimProducer package